packages/content-source-maps/src/graphql/encodeGraphQLResponse.ts

```diff
@@ -0,0 +1,91 @@
+import { has, get } from 'json-pointer';
+import type { GraphQLResponse } from '../types.js';
+import {
+  clone,
+  createSourceMapMetadata,
+  encodeField,
+  isBuiltinNamespace,
+  isSupportedWidget,
+} from '../utils.js';
+
+export const encodeGraphQLResponse = (
+  originalGraphqlResponse: GraphQLResponse,
+  targetOrigin?: 'https://app.contentful.com' | 'https://app.eu.contentful.com',
```

Comment:
@YvesRijckaert do we still need to add localhost here for testing locally or is that fine?

Reply:
Good question, `targetOrigin` here is only relevant for Vercel visual editor, it will open then the correct contentful link
So I don't think it's relevant to have localhost here 🤔